commit 1f3e9d4811b846974d8f882ce00697e889a80788
Author: Sage Weil <sweil@redhat.com>
Date:   Mon Dec 17 14:22:33 2018 -0600

    mgr/PyModule: populate ModuleOptions and expose to mon
    
    Populate the ModuleOptions from MODULE_OPTIONS fields and pass it to the
    mon for inclusion in the MgrMap.
    
    Signed-off-by: Sage Weil <sage@redhat.com>

diff --git a/src/mgr/MgrStandby.cc b/src/mgr/MgrStandby.cc
index 70e504c7010..8522fcd7d20 100644
--- a/src/mgr/MgrStandby.cc
+++ b/src/mgr/MgrStandby.cc
@@ -195,6 +195,7 @@ void MgrStandby::send_beacon()
     info.name = module->get_name();
     info.error_string = module->get_error_string();
     info.can_run = module->get_can_run();
+    info.module_options = module->get_options();
     module_info.push_back(std::move(info));
   }
 
